webapp-django/crashstats/topcrashers/jinja2/topcrashers/topcrashers.html

```diff
@@ -164,18 +164,39 @@
                                 <input type="hidden" class='ajax-signature' name="ajax-signature-1" value="{{ crash.signature }}" />
                             </div>
                             <div class="signature-icons">
+                                {% if crash.startup_count > 0 %}
+                                {% if crash.startup_count == crash.count %}
```

Comment:
Because of the ``else`` that follows, and the fact that I thus don't have to write ``startup_count > 0`` twice.